src/js/components/modals/EditAppModalComponent.jsx

```diff
@@ -73,7 +73,8 @@ var EditAppModalComponent = React.createClass({
 
   onRequestAppVersionError: function () {
     var props = this.props;
-    DialogActions.alert(`Could not fetch app '${props.appId}' version for editing: ${props.appVersion}`);
+    DialogActions.alert(`Could not fetch app '${props.appId}' version`
```

Comment:
The decision on the ternary operator notation informed the leading plus I used here.